guides/hosting/performance/performance-tweaks.md

```diff
@@ -0,0 +1,208 @@
+# Performance Tweaks
+
+Shopware is a platform for many different projects. It needs to handle a broad range of load characteristics and environments. That means that the default configuration is optimized for the best out-of-the-box experience. But there are many opportunities to increase the performance by fitting the configuration to your needs.
+
+## Strong CPU
+
+For the server setup, pay special attention to the speed of the CPU. This applies to all servers (app, SQL, Elasticsearch, Redis). Usually, it is more optimal to choose a slightly stronger CPU instead of many weaker ones. But this has to be determined more precisely depending on the project and load. Experience has shown so far that systems with powerful CPUs finish processes faster and can therefore release the resources sooner.
+
+## Redis
+
+We recommend setting up at least four Redis servers for the following resources:
+- Session + cart - [Read more](../performance/session.md)
+- cache.object - [Read more](../performance/caches.md#example-replace-some-cache-with-redis)
+- Lock + Increment storage - [Read more](../performance/increment.md)
+- Enqueue - [Read more](../infrastructure/message-queue.md#transport-redis-example)
+
+Instead of setting up a Redis server for `enqueue`, you can also work directly with [RabbitMQ](../infrastructure/message-queue.md#transport-rabbitmq-example)
+
+The PHP Redis extension should be preferred to be installed on the machine as of Predis to use a persistent Redis connection. Persistent connection opens per FPM process a Redis connection and uses it for all requests, which can help in high load scenarios as each request doesn't have to open and close a connection and may hit the system max open sockets
+
+When a Redis cluster is in usage, the `php.ini` setting `redis.clusters.cache_slots=1` should be set to skip the cluster node lookup on each connection 
+
+## Logging
+
+Set the log level of monolog to `error` to reduce the amount of logged events. Also, limiting the `buffer_size` of monolog prevents memory overflows for long-lived jobs:
+
+```yaml
+monolog:
+    handlers:
+        main:
+            level: error  
+            buffer_size: 30
+        business_event_handler_buffer:
+            level: error
+```
+
+The `business_event_handler_buffer` handler logs flows. Setting it to `error` will disable the logging of flow activities that succeed
+
+## Filesystem
+
+In a multi-app-server system, manage specific directories over a shared filesystem. This includes assets as well as theme files and private and public filesystems. The recommendation is an S3 compatible bucket instead of NFS shares or GCP backend.
```

Comment:
Do we prefer S3 over GCP? Or Do we prefer S3 and GCP over NFS? This is unclear from this sentence